modules/gapi/test/common/gapi_video_tests_inl.hpp

```diff
@@ -88,6 +89,46 @@ TEST_P(BuildPyr_CalcOptFlow_PipelineTest, AccuracyTest)
     compareOutputsOptFlow(outOCV, outGAPI);
 }
 
+#ifdef HAVE_OPENCV_VIDEO
+TEST_P(BackgroundSubtractorTest, AccuracyTest)
+{
+    initTestDataPath();
+
+    cv::gapi::video::BackgroundSubtractorType opType = std::get<0>(typeAndThreshold);
+    double threshold = std::get<1>(typeAndThreshold);
```

Comment:
I agree that you have to use `tuple` here as threshold depends on subtractor's type.
But it is better to predefine two variables and then use `std::tie()` instead of using `std::get<>`
I mean,
```
    cv::gapi::video::BackgroundSubtractorType opType = -1;
    double threshold = -1;
    std::tie(opType, threshold) = typeAndThreshold;
```